safari/Refined GitHub.xcodeproj/project.pbxproj

```diff
@@ -264,16 +240,8 @@
 		E3919AC127E6107E009C4956 /* Resources */ = {
 			isa = PBXGroup;
 			children = (
-				E3919AEA27E61192009C4956 /* background.js */,
-				E3919AED27E61192009C4956 /* icon.png */,
+				1F9945312A07DB5E005BC236 /* assets */,
```

Comment:
1. I checked out this PR
2. Got `codesign` error again
3. Renamed this subfolder to `assets`
4. It builds now, both iOS and macOS

<img width="235" alt="Screenshot 7" src="https://github.com/refined-github/refined-github/assets/1402241/a36a1672-7d9e-4fef-b92a-45efeba66a3c">